src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.2.0-alpha1
+
+Released 2021-Jul-23
+
 * Add Metrics support.
 
 * Removes .NET Framework 4.5.2 support. The minimum .NET Framework
```

Comment:
Just noticed that we didn't mention 4.6 here, is this intentional?

Reply:
i think @utpilla commented same in another PR. I initially thought the 2nd sentence was sufficient, but now I think its best to mention removed net452,net46 support :)

Reply:
😸